notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>REGION should not be hardwired</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Edited. Defined REGION variable inside an if statement and not hardcoded.